source/core_mqtt.c

```diff
@@ -1002,32 +1002,38 @@ static MQTTStatus_t sendPublishAcks( MQTTContext_t * pContext,
 static MQTTStatus_t handleKeepAlive( MQTTContext_t * pContext )
 {
     MQTTStatus_t status = MQTTSuccess;
-    uint32_t now = 0U, keepAliveMs = 0U;
+    uint32_t now = 0U, packetTxTimeoutMs = 0U, packetRxTimeoutMs = 0U;
```

Comment:
packetRxTimeoutMs is unused.
```suggestion
    uint32_t now = 0U, packetTxTimeoutMs = 0U;
```